package.json

```diff
@@ -90,6 +90,7 @@
     "tailwindcss": "^2.2.4"
   },
   "dependencies": {
+    "ember-autofocus-modifier": "^3.0.0",
```

Comment:
@galfert This was probably an accident, and should be a dev dependency, right?

Reply:
Oh, that's weird. I installed it with `ember install` and that usually adds it as a dev dependency. I'll change it.